### DISCUSSION TITLE
Include compilled browser bundles in package

### DISCUSSION THREAD

**Discussion by trymeouteh** [ajv-validator/ajv#2371] [Ideas]
I was unable to compile ajv to create a JS file to use in the browser. I had to use a CDN which is fine in most cases, but I wanted to have ajv stored locally.

Please include browser UMD buids in the package, or even consider creating a new repo and npm package that is ajv compilled for the browser and is easy to plug in play.

**Comment by GabenGar** [comment]
What do you mean by "compile AJV"?

**Reply by trymeouteh** [reply]
There is no UMD bundle available for Ajv

https://github.com/ajv-validator/ajv/issues/1744
